src/v2/Apps/Order/__tests__/OrderApp.jest.tsx

```diff
@@ -23,13 +23,21 @@ import { FarceRedirectResult } from "found/server"
 import { DateTime } from "luxon"
 import { Environment, RecordSource, Store } from "relay-runtime"
 import { GlobalData } from "sharify"
-
-jest.mock("react-stripe-elements", () => ({
-  CardElement: () => jest.fn(),
-  Elements: ({ children }) => children,
-  StripeProvider: ({ children }) => children,
-  injectStripe: () => jest.fn(),
-}))
+import { mockStripe } from "v2/DevTools/mockStripe"
+
+jest.mock("@stripe/stripe-js", () => {
+  let mock = null
+  return {
+    loadStripe: () => {
+      if (mock === null) {
+        mock = mockStripe()
+      }
+      return mock
+    },
+    _mockStripe: () => mock,
+    _mockReset: () => mock = mockStripe(),
+  }
+})
```

Comment:
(thought) Have noticed this pattern a few times -- could we move it to the root level `__mocks__` folder to reduce duplication?

Reply:
Possibly in a future PR, these tests are riddle with seemingly random mocks and mock reset calls and every time I attempted to extract the mocks everything broke. 😬